wdl/EvidenceQC.wdl

```diff
@@ -169,5 +194,66 @@ workflow EvidenceQC {
     File bincov_matrix = MakeBincovMatrix.merged_bincov
     File bincov_matrix_index = MakeBincovMatrix.merged_bincov_idx
     File bincov_median = MedianCov.medianCov
+
+    File qc_table_csv = MakeQcTable.qc_table_csv
   }
 }
+
+task MakeQcTable {
+  input {
+    String? prefix
+
+    File? ploidy_plots
+    File bincov_median
+    File WGD_scores
+    Float? melt_insert_size
+
+    File? manta_qc_low
+    File? manta_qc_high
+    File? melt_qc_low
+    File? melt_qc_high
+    File? wham_qc_low
+    File? wham_qc_high
+    File? scramble_qc_low
+    File? scramble_qc_high
+
+    String sv_pipeline_docker
+    RuntimeAttr? runtime_override
+  }
+
+  output {
+    File qc_table_csv = "${prefix}_table.csv"
```

Comment:
TSV please

Reply:
Actually this table is going to be big, let's gzip it

Reply:
How big do you think this can be? @shadizaheri is currently using pandas dataframe for simplicity keeps the table in memory, if this will be big, probably we should use a different approach that does not keep the data in the memory.

Reply:
At a batch level, I think it's fine to use pandas. Eventually on the cohort level (not within this workflow) we might run into issues with very large datasets.

Reply:
I don't have a size estimate, so you could see how big it gets for 156 ref panel samples and then x5 for a larger batch

Reply:
I think @shadizaheri has a memory usage estimate for this.